recipe/meta.yaml

```diff
@@ -26,8 +22,10 @@ requirements:
     - python
     - pip
     - cython
-    - _libarchive_static_for_cph ==3.4.3
+    - _libarchive_static_for_cph ==3.6.1
```

Comment:
big jump

Reply:
Hmm, it's only to address CVEs https://github.com/AnacondaRecipes/conda-package-handling-feedstock/issues/5